views/js/pciCreator/dev/likertMy/creator/tpl/propertiesForm.tpl

```diff
@@ -0,0 +1,21 @@
+<div class="panel">
+    <label for="level">{{__ "Level"}}</label>
+    <span class="icon-help tooltipstered" data-tooltip="~ .tooltip-content:first" data-tooltip-theme="info"></span>
+    <span class="tooltip-content">{{__ "Scale size"}}</span>
+    <select name="level" class="select2" data-has-search="false">
+        {{#each levels}}
+        <option value="{{@key}}" {{#if selected}}selected="selected"{{/if}}>{{label}}</option>
+        {{/each}}
+    </select>
+</div>
+<div class="panel">
+    <label>
+        <input name="navigationLock" type="checkbox" {{#if navigationLock}}checked="checked"{{/if}}/>
+        <span class="icon-checkbox"></span>
+        {{__ "Navigation lock"}}
+    </label>
+    <span class="icon-help tooltipstered" data-tooltip="~ .tooltip-content:first" data-tooltip-theme="info"></span>
+    <span class="tooltip-content">
+        {{__ 'If this box is checked the test runner navigation to be disabled until the test-taker gave an answer to it.'}}
```

Comment:
**quibble:** Not exactly. The RFE-223 proposed several possibilities, and the one that was chosen is not disabling the navigation. Instead, it blocks it, the user can still click the navigation button, but when doing so, a message is displayed and the navigation is canceled.